src/Timeline.js

```diff
@@ -196,10 +222,35 @@ export default class Timeline extends Axis {
       @chainable
   */
   render(callback) {
+    const {height, y} = this._position;
+    this._d3Scale = scaleTime().domain(this._domain.map(date)).range([0, this._width]);
 
-    super.render(callback);
+    const tickFormat = this._d3Scale.tickFormat();
 
-    const {height, y} = this._position;
+    // Measures size of ticks
+    const textData = this._d3Scale.ticks().map((d, i) => {
```

Comment:
use a `reduce` instead of a `map` here to calculate `ticksWidth` without having to loop through the ticks a second time (and so you can get rid of the reducer below). `textData` is not needed